apps/mobile-wallet/src/navigation/RootStackNavigation.tsx

```diff
@@ -140,12 +140,11 @@ export default RootStackNavigation
 const AppUnlockModal = () => {
   const dispatch = useAppDispatch()
   const isWalletUnlocked = useAppSelector((s) => s.wallet.isUnlocked)
+  const biometricsRequiredForAppAccess = useAppSelector((s) => s.settings.usesBiometrics)
   const navigation = useNavigation<NavigationProp<RootStackParamList>>()
   const { triggerBiometricsAuthGuard } = useBiometricsAuthGuard()
   const { t } = useTranslation()
 
-  const [isAuthModalVisible, setIsAuthModalVisible] = useState(false)
```

Comment:
There is no reason to maintain this value, we can calculate it (see `visible` prop of following modal)